tags/views.py

```diff
@@ -34,12 +34,13 @@
 
 
 def tags(request, multiple_tags=None):
-
+    
     if multiple_tags:
         multiple_tags = multiple_tags.split('/')
     else:
         multiple_tags = []
-    multiple_tags = sorted([x for x in multiple_tags if x])
+    #Make all tags lower-cased and unique to get a case-insensitive search filter and shortened browse url
+    multiple_tags = sorted(set([x.lower() for x in multiple_tags if x]))
```

Comment:
No need to for the list comprehension if we turn it into a set anyways
```suggestion
    multiple_tags = sorted({x.lower() for x in multiple_tags if x})
```